nemo/collections/multimodal/models/text_to_image/stable_diffusion/ldm/autoencoder.py

```diff
@@ -316,6 +317,7 @@ def __init__(
         ignore_keys=[],
         image_key="image",
         colorize_nlabels=None,
+        from_NeMo=False,
```

Comment:
use `from_nemo` lowercase

Reply:
I've tried to stay consistent with the format in `nemo/collections/multimodal/modules/stable_diffusion/diffusionmodules/openaimodel.py` (L1220).